e2e/playwright/tests/command/me.spec.ts

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+import {expect, test} from '@e2e-support/test_fixture';
+
+import {messages} from '../../support/constants';
+import {getGithubBotDMPageURL, waitForNewMessages} from '../../support/utils';
+import {
+    getBotTagFromPost,
+    getPostAuthor,
+} from '../../support/components/post';
+
+const mmGithubHandle = 'MM-Github-Plugin';
+
+export default {
+    connected: () => {
+        test.describe('/github me', () => {
```

Comment:
Is this test dependent on any other test to be able to complete the pre-requisite steps of connecting their github account? I am sorry, but I am not seeing any references here.

Reply:
unfortunately, tests are dependent and ran serially in a specific order specified in test.list.ts

in the future, we will explore better approaches to prepare the entire env for each test, due to the complexity, the first full iteration is like this.